pkg/core/resources/apis/mesh/proxytemplate_validator.go

```diff
@@ -23,12 +23,20 @@ func init() {
 
 func (t *ProxyTemplateResource) Validate() error {
 	var verr validators.ValidationError
-	verr.Add(validateImports(t.Spec.Imports))
-	verr.Add(validateResources(t.Spec.Resources))
 	verr.Add(validateSelectors(t.Spec.Selectors))
+	verr.AddError("conf", validateConfig(t.Spec.Conf))
 	return verr.OrNil()
 }
 
+func validateConfig(conf *v1alpha1.ProxyTemplate_Conf) validators.ValidationError {
+	var verr validators.ValidationError
+	if conf != nil {
+		verr.Add(validateImports(conf.Imports))
```

Comment:
Let's do `validateImports(conf.GetImports())` instead of wrapping `if conf != nil {`